.github/workflows/screenshots.yml

```diff
@@ -16,6 +16,8 @@ jobs:
     env:
       PERCY_PARALLEL_NONCE: ${{ github.run_id }}-${{ github.run_attempt }}
       PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+      PERCY_POSTINSTALL_BROWSER: false
+      PUPPETEER_SKIP_DOWNLOAD: true
```

Comment:
Prevents `npm ci` from downloading ~140MB of browser when running manually:
https://github.com/alphagov/govuk-frontend/actions/workflows/screenshots.yml